docs/release-notes/RELEASE-NOTES-0.17.0.md

```diff
@@ -1,9 +1,10 @@
 # About this release
 
-[This release][this-release] is focused on TODOFILLMEIN.
+[This][this-release] is focused on the first batch of work to support Bitcoin-NG in Aeternity.
 It:
 * Stores the chain state in the database in a more compact way. This affects the persisted chain.
-* Does this.
+* Changes the way we build chain, by electing a leader and introducing Key and Micro blocks. It is followed by new candidates generation modules, new serialized formats and enhanced sync layer.
+* Manages node keys in more secure way
```

Comment:
Could you please point me to what change post v0.16.0 refers to?

Reply:
it is Tobias' sensitive flag and aec_keys refactoring

Reply:
@lucafavatella I see it's included in 0.16.0 - fixing